example/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "ExampleApp",
+  "name": "exampleApp",
   "version": "0.0.1",
   "private": true,
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "lint": "eslint .",
-    "pod": "(rm -rf ios/Pods && cd ios/ && pod install --repo-update)",
+    "pod": "(rm -rf ios/Pods && cd ios/ && pod install --repo-update && pod update hermes-engine --no-repo-update)",
```

Comment:
Obviously less than ideal to have to do this, but it does appear to remedy with the only byproduct being downloading the tarball twice